### ISSUE TITLE
Add properties for modifiying model armature transformation in world

### ISSUE THREAD

**Issue by timing1337** [dtzxporter/cast#79]

**Comment by dtzxporter** [comment]
I'm not sure what you mean, can you give an example?

**Comment by timing1337** [comment]
basically the ability to change this directly through properties preferably in Skeleton node
![Screenshot 2024-12-21 214901](https://github.com/user-attachments/assets/65b3d0f2-c716-4404-80c7-d03174fd0b03)

**Comment by dtzxporter** [comment]
Ahh so transform properties on a per-model basis?

**Comment by timing1337** [comment]
Yeah exactly! My reason for this is using cast as a storage for scene with multiple models. Though I've found a better approach to this, this could be neat for scene with pretty small number of models.

**Comment by dtzxporter** [comment]
Yeah, sounds like a reasonable feature! For large scenes, I'd recommend instance nodes.

**Comment by dtzxporter** [comment]
Going to implement this soon.

**Comment by dtzxporter** [comment]
Now I said soon, but where is the fun in that!
